src/tennissimulator/match/EmptyDatabaseException.java

```diff
@@ -0,0 +1,9 @@
+package tennissimulator.match;
+
+public class EmptyDatabaseException extends RuntimeException {
```

Comment:
Maybe introduce separate package "exception" and place both exceptions there?

Reply:
I browsed internet for naming of package with custom exceptions, but find out some answers that it is good idea to keep exception classes in the same package with classes, which handle these exceptions. If it is normal to use separate package, I will create it.